parsons/google/google_cloud_storage.py

```diff
@@ -311,7 +312,10 @@ def upload_table(self, table, bucket_name, blob_name, data_type="csv", default_a
         blob = storage.Blob(blob_name, bucket)
 
         if data_type == "csv":
-            local_file = table.to_csv()
+            if isinstance(table.table, petl.io.csv_py3.CSVView):
```

Comment:
Just want to make sure I understand this (and may make sense to add this as a comment, so future people can understand this) - does the check that `table.table` is instance of `CSVView` mean that `table.table` has not been transformed? Otherwise how would we make sure the table hasn't been altered and we're not using an out-of-date file?

Reply:
Yes that's right! Once you do any transformation on a parsons Table, the Table.table petl object will no longer be a CSVView. I'll add that as a comment